tests/smoke/fake-amo-server.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
```

Comment:
It would be nice to stick a comment at the top of this file linking to http://addons-server.readthedocs.io/en/latest/topics/api/signing.html